lib/lint.js

```diff
@@ -0,0 +1,325 @@
+'use babel';
```

Comment:
If you remove this directive, and replace `export function provideLinter()` with `module.exports = {provideLinter};` instead, we can avoid the transpilation step and quicken package startup time.

Reply:
Done